_includes/tutorials/connect-sink-timestamp/ksql/markup/dev/create-sink-connector.adoc

```diff
@@ -0,0 +1,19 @@
+Now deploy a JDBC sink connector with the code below. Notice no transforms will be necessary to the `eventTime` column because the column is already a supported `TIMESTAMP` type.
```

Comment:
For your consideration: maybe mention that the sink connector writes to the database.  I know most readers would get this, but a brand new user could miss something.

Reply:
Good point. I added a comment to address this in my latest commit. https://github.com/confluentinc/kafka-tutorials/pull/760/commits/9904d2ac9ffb1d0894c431caf88c786ba19ad091